ddocs/users-db/users/views/users_by_contact_id/map.js

```diff
@@ -0,0 +1,5 @@
+function(doc) {
+  if (doc.contact_id) {
+    emit(doc.contact_id);
```

Comment:
A common pattern that we've used in other views that index related properties is to emit an array, like: 
```
function(doc) {
  if (doc.contact_id) {
    emit(['contact_id', doc.contact_id]);
  }
  if (doc.facility_id) {
    emit(['facility_id', doc.facility_id]);
  }
}
```

I think adding multiple views makes sense when the view code is very different. One other advantage is that you could emit both facility and contact and potentially open the possibility to query for both. For this, I think a single view can work ok. What do you think?